### ISSUE TITLE
I would like to ask you to remove my source codes from your repository

### ISSUE THREAD

**Issue by V0ID1337x** [Sh4d0v/Gothic-Online#1]

**Comment by Sh4d0v** [comment]
@V0ID1337x Dlaczego? Czy nie właśnie taki był ostateczny zamiar? :)
Jeśli to Twoja ostateczna decyzja - daj znać.

**Comment by V0ID1337x** [comment]
@Sh4d0v tak, to ostateczna decyzja. Próbowałem się z Tobą wcześniej skontaktować, ale po kolejnym spamie z pytaniami o ten source postanowiłem po prostu otworzyć nowy issue.
